eng/pipelines/aggregate-reports.yml

```diff
@@ -168,6 +168,27 @@ stages:
                 -WorkingDirectory '$(Build.ArtifactStagingDirectory)'
                 -NupkgFilesDestination 'nupkgFiles'
 
+          - task: AzureCLI@2
+            displayName: Azure CLI Login
+            inputs:
+              azureSubscription: azure-sdk-apiscan
+              scriptType: pscore
+              scriptLocation: inlineScript
+              addSpnToEnvironment: true
+              inlineScript: |
+                az --version
+                az account show
+                Write-Host "##vso[task.setvariable variable=ARM_CLIENT_ID;issecret=true]$($env:servicePrincipalId)"
+                Write-Host "##vso[task.setvariable variable=ARM_TENANT_ID;issecret=true]$($env:tenantId)"
+                Write-Host "##vso[task.setvariable variable=ARM_OIDC_TOKEN;issecret=true]$($env:idToken)"
+
+          - pwsh: |
+                # Need to re-login with the az login so that it presists and can be used in the APISca task
+                az login --service-principal -u $(ARM_CLIENT_ID) --tenant $(ARM_TENANT_ID) --allow-no-subscriptions --federated-token $(ARM_OIDC_TOKEN)
+                az --version
+                az account show
```

Comment:
```suggestion
                az account show -o json
```

Reply:
What does `-o json` by us? I just tried locally and it appears to dump the same string to the console.

Reply:
I think in the pipeline it usually defaults to json so the output is the same, but for example in my terminal it usually defaults to tsv, which omits the subscription id. Specifying json explicitly means we always get the intended output.